lib/api/openid_connect/authorization_point/endpoint_start_point.rb

```diff
@@ -17,7 +17,7 @@ def handle_response_type(req, _res)
         end
 
         def replace_profile_scope_with_specific_claims(req)
-          profile_claims = %w(sub aud name nickname profile picture)
+          profile_claims = %w[sub name nickname profile picture gender birthdate locale updated_at]
```

Comment:
We don't need as specific scopes for the profile, a single `profile` scope should be enough, or maybe a second `private_profile` one if you insist

Reply:
I think it's important to separate public and profile fields here.

Reply:
Nvm, I misunderstood what this is doing